test/inductor/test_torchinductor.py

```diff
@@ -4617,6 +4617,17 @@ def fn(a):
 
         self.common(fn, (torch.randn([3, 3, 6, 12]),))
 
+    def test_upsample_bilinear2d_uint8(self):
```

Comment:
NO need to add an inductor test. Decompositions are tested through opinfos, so just add a new sample to the relevant OpInfo.

Reply:
There is a test case in https://github.com/pytorch/pytorch/blob/faea6f2c7a4d6e6e7ebe21752eaa7b0ad05afa79/torch/testing/_internal/common_methods_invocations.py#L4596, but in `reference_inputs_func` instead of `sample_inputs_func`. Not sure if I should move that to `sample_inputs_func` or make the tests use `reference_inputs` instead of `sample_inputs`.

Reply:
We can:
1. Move that test case to sample inputs in this PR
2. Run the decomposition tests (the forward, not the backward) on both sample inputs and reference inputs. Let's do this in a different PR as we are surely going to uncover bugs when doing so.